src/main/java/com/csc366/project/entity/Owner.java

```diff
@@ -0,0 +1,95 @@
+package com.csc366.project.entity;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.StringJoiner;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="owner")
+public class Owner {
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private String firstName;
+    private String lastName;
+
+    // @OneToMany(mappedBy = "address")    //is this correct?
+    // private Set<Location> locations = new HashSet<Location>();
```

Comment:
If we support an owner being able to own multiple locations, then yes, this is correct. It should be mapped by `location` though, not `address`

Reply:
If you're still seeing errors when you uncomment this, you might also try fiddling with the `owner` property's annotations in the `Location` class